docs/application/dotnet/guides/uix/sticker.md

```diff
@@ -0,0 +1,352 @@
+
+            return stickerData;
+        }
+        catch (Exception e)
+        {
+            /* Error handling */
+        }
+    }
+    ```
+
+<a name="set_json"></a>
+### Set sticker information in JSON file
+
+The following JSON file shows how to set the sticker information to be stored in the Sticker database:
+
+    ```
+    {
+        "sticker":
+        [
+            {
+                "type" : 1,  /* Sets the URI type of the sticker */
+                "uri" : "/res/face/slightly_smiling_face.png",  /* Sets the URI of the sticker */
+                "keyword" : ["face", "smile"],  /* Adds a keyword of the sticker to the list */
+                "group" : "face",  /* Sets the group name of the sticker */
+                "thumbnail" : "/res/thumbnail/face/slightly_smiling_face.png",  /* Sets the thumbnail local path of the sticker */
+                "description" : "A yellow face with simple, open eyes and a thin, closed smile."  /* Sets the description of the sticker */
+            },
+            {
+                "type" : 1,
+                "uri" : "/res/face/heart_eyes.png",
+                "keyword" : ["face", "love", "adoration"],
+                "group" : "face",
+                "thumbnail" : "/res/thumbnail/face/heart_eyes.png",
+                "description" : "A yellow face with an open smile, sometimes showing teeth, and red, cartoon-styled hearts for eyes."
+            },
+            ...
+            {
+                ...
+            }
+        ]
+    }
+    ```
+
+<a name="insert_sticker"></a>
+### Insert sticker information
+
+You can use the following methods to store sticker information in the Sticker database:
+
+- To store one sticker information, use `InsertData(StickerData data)`:
+
+    ```csharp
+    void InsertStickerData(StickerData stickerData)
+    {
+        try
+        {
+            StickerProvider.InsertData(stickerData);
+        }
+        catch (Exception e)
+        {
+            /* Error handling */
+        }
+    }
+    ```
+
+- To store multiple sticker information, use `InsertData(string jsonPath)`:
+
+    ```csharp
+    async Task InsertStickerData(string jsonPath)
+    {
+        try
+        {
+            await StickerProvider.InsertData(jsonPath);
+        }
+        catch (Exception e)
+        {
+            /* Error handling */
+        }
+    }
+    ```
+
+- To store sticker information without using `InsertData(StickerData data)` and `InsertData(string jsonPath)`, add the following metadata to the `tizen-manifest.xml` file:
+
+  ```
+  <category name="http://tizen.org/category/sticker"/>
+  <!-- Sets the JSON local path -->
+  <metadata key="http://tizen.org/metadata/sticker" value="/res/json/sticker.json"/>
+  ```
+
+  If you set the path of the JSON file to the value in metadata, sticker information of the JSON file is automatically stored when the Sticker provider application is installed.
+
+  > [!NOTE]
+  >
```

Comment:
Please remove the extra line after the alert.
```suggestion
```

Reply:
done